api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -56,7 +63,7 @@ export type Template = keyof typeof emailTemplates;
 const sendEmail = async (
   template: Template,
   emailAddress: string,
-  config: SaveAndReturnNotifyConfig | EmailSubmissionNotifyConfig
+  config: SaveAndReturnNotifyConfig | EmailSubmissionNotifyConfig | InviteToPayNotifyConfig
```

Comment:
it'd be nice to think more intentionally about these types and what can be shared going forward: eg is there a `MinimumNotifyPersonalisation` type that we could then extend for domain-specific templates ?

i took a quick pass at this and introduced some unexpected errors, so just settled on introducing a discrete new type here to keep the scope of changes in check. but one to come back to soon!